src/plugins/workspace/public/components/workspace_initial/workspace_initial.tsx

```diff
@@ -40,10 +40,14 @@ export const WorkspaceInitial = () => {
   const isDarkTheme = uiSettings.get('theme:darkMode');
   const backGroundUrl = isDarkTheme ? BackgroundDarkSVG : BackgroundLightSVG;
 
-  const noAdminToolTip = i18n.translate('workspace.initial.card.createWorkspace.toolTip', {
-    defaultMessage:
-      'Contact your administrator to create a workspace or to be added to an existing one.',
-  });
+  const noAdminToolTip = (
+    <>
+      {i18n.translate('workspace.initial.card.createWorkspace.toolTip', {
+        defaultMessage:
+          'Contact your administrator to create a workspace or to be added to an existing one.',
+      })}
+    </>
+  );
```

Comment:
OuiToolTip's `content` can take strings as part of RactNode.
```suggestion
  const noAdminToolTip = i18n.translate('workspace.initial.card.createWorkspace.toolTip', {
      defaultMessage:
        'Contact your administrator to create a workspace or to be added to an existing one.',
  });
```

Reply:
Thanks, have updated.